cornedbeef/build.gradle

```diff
@@ -30,39 +29,29 @@ dependencies {
     testImplementation 'org.mockito:mockito-core:2.28.2'
 }
 
-publishing.publications {
-    aar(MavenPublication) {
-        groupId "TouchType"
-        version = android.defaultConfig.versionName
-        artifactId project.getName()
+publishing {
+    repositories {
 
-        // Tell maven to prepare the generated "*.aar" file for publishing
-        artifact("$buildDir/outputs/aar/${project.getName()}-release.aar")
-    }
-}
+        def user=project.properties['publish_user'] ?: "admin"
+        def pass=project.properties['public_pass'] ?: "password"
+        def url=project.properties['public_url'] ?: "http://localhost:8081/artifactory"
```

Comment:
It doesn't matter because nothing will work if we use defaults, but we could remove all references to Artifactory.